src/client/src/App.js

```diff
@@ -23,7 +23,7 @@ function App() {
           <Route path="/users/send-forgot-password-email" component={SendForgotPasswordEmail} />
           <Route path="/users/change-password" component={ForgotPassword} />
           <Route path="/index" render={(props) => <IndexPage {...props} userId='5f53bae924e27d18580d4036'/>} />
-          <Route path="/" component={IndexPage} />
+          {/* <Route path="/" component={IndexPage} /> */}
```

Comment:
I don't know how to fix it but we need to have a route to the landing pages in the Node.JS as well...

Reply:
Will look into it later